<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOver</class>
 <widget class="QWidget" name="GameOver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameOver | Pacman</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../imgresources.qrc">
    <normaloff>:/img/pacman/1.png</normaloff>:/img/pacman/1.png</iconset>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/background);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logo_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/pacman_logo);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>130</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ENTER YOUR NAME,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>HIGH SCORER!</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>289</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>61</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 128);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goback_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>530</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 128);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>GO BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="initial_gif_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="yourscore_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>YOUR SCORE =</string>
   </property>
  </widget>
  <widget class="QLabel" name="editable">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../imgresources.qrc"/>
 </resources>
 <connections/>
</ui>
